rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionInterpreter.java

```diff
@@ -45,9 +46,20 @@ public SolutionInterpreter( Solution solution, boolean appendWon )
         this.command = null;
         this.untilCount = -1;
     }
+    
+    public static SolutionInterpreter getNothingPlaying()
+    {
+        SolutionInterpreter nothingPlaying = new SolutionInterpreter( SolutionParser.parse( "1" ) );
+        nothingPlaying.emptySteps = true;
+        return nothingPlaying;
+    }
```

Comment:
If you still want getNothingPlayer I suggest a rename to createNothingPlayer. Get is both a horrible word and usually used to mean access a member variable.